components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/utils/ExportUtils.java

```diff
@@ -0,0 +1,175 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Modify the year

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/9533/commits/da55b408c701b8408efa0ae77d53b5fdb89f49a1